src/mgr/PyModule.cc

```diff
@@ -36,6 +36,11 @@ std::string PyModule::mgr_store_prefix = "mgr/";
 
 // Courtesy of http://stackoverflow.com/questions/1418015/how-to-get-python-exception-text
 #define BOOST_BIND_GLOBAL_PLACEHOLDERS
+// Fix instances of "'BOOST_PP_ITERATION_02' was not declared in this scope; did you mean 'BOOST_PP_ITERATION_05'"
+// and related macro error bullshit that spans 300 lines of errors
+//
+// Apparently you can't include boost/python stuff _and_ have this header defined
```

Comment:
are there any docs we maybe able reference for this statement? (so that we have some context for this long term?)

Reply:
Sorry, I have no idea. I didn't write this statement, it's from the Arch Linux repository. I've never used boost::python (because I'm neither a Boost nor a Python fan). This Ceph build failure confuses me, and only this patch fixes it.